app/main.py

```diff
@@ -1,6 +1,38 @@
+import os
+from datetime import datetime as dt
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+BASE_URL = "https://api.weatherapi.com/v1/current.json"
+API_KEY = os.getenv("API_KEY")
+CITY = "Paris"
```

Comment:
The same goes for the `CITY` constant. According to the Python style guide, constants should be named in uppercase. So, you should rename `CITY` to `FILTERING`.